class/Utils/Utils.php

```diff
@@ -53,5 +54,31 @@ static function clear_featured_posts()
   {
     delete_metadata("post", 0, "post_is_featured_on_passle_page", "", true);
     delete_metadata("post", 0, "post_is_featured_on_post_page", "", true);
+    self::clear_featured_posts_cache();
+  }
+
```

Comment:
clearing the cache here seems a bit odd, maybe this should be done somewhere else? 
looking at the code i think that delete_metadata should have done the trick within WP, is the problem that when posts sync, we recover the old posts from the cache that leads you to have to do this?